### ISSUE TITLE
Expose test duration in `results.yaml`

### ISSUE THREAD

**Issue by thrix** [psss/tmt#456]
It would be nice if we could provide test time in xunit and if we would not need text output of `tmt` because of it.

See:

https://pagure.io/fedora-ci/general/issue/157

For details.

**Comment by thrix** [comment]
Feels like it could be called runtime and be in seconds, similarly to:

https://pagure.io/standard-test-roles/pull-request/393#request_diff

**Comment by thrix** [comment]
I will try to prepare a patch

**Comment by psss** [comment]
@thrix, we have already outlined the duration in the [execute step spec](https://tmt.readthedocs.io/en/latest/spec/steps.html#execute):

```yaml
/test/two:
    result: OUTCOME
    log: PATH
    duration: DURATION
```

The DURATION is an optional section stating how long did the test run. Its value is in the `hh:mm:ss` format.

**Comment by dkarpele** [comment]
I'll look at it

**Comment by lukaszachy** [comment]
Closed in #610 . 
Apologies for messing autoclosing.
